docs/learn/using-zeitgeist-markets.md

```diff
@@ -300,6 +300,45 @@ dispute for the token which the market eventually resolved to receive their
 stakes tokens back, while the stake of those who staked for other tokens are
 slashed.
 
+### Bonds
+
+There are four types of bonds utilized by prediction markets on Zeitgeist:
+_creation_, _oracle_, _outsider_ and _dispute_ bonds. Bonds are deemed _settled_
+when they are entirely unreserved or slashed. In the event of a market's
+destruction by the Advisory Committee - a course of action only considered for
+markets in severe violation of our policies - all associated bonds will be
+slashed. Slashed funds are subsequently transferred to the treasury.
+
+- The _creation bond_ is reserved by the market creator at the time of market
+  creation. Its size varies depending on whether the market's creation type is
+  advised or permissionless, with the latter requiring a larger amount. If the
+  market is advised, the creation bond is unreserved upon market approval or if
+  the market concludes without approval. If the market is rejected, a portion of
+  the bond is slashed, and the remaining part is unreserved. The specific
+  portion to be slashed is determined by the `AdvisoryBondSlashPercentage`
+  parameter. Conversely, if the market is created permissionlessly, the creation
+  bond is unreserved upon the market's resolution. Regardless of the scenario,
+  the creation bond serves to ensure the market creator's appropriate conduct.
+
+- The _oracle bond_ is reserved by the market creator during market creation. It
+  is unreserved upon the market's resolution if the oracle provided an honest
+  report, confirmed as true by dispute or undisputed. If the oracle failed to
+  provide a report or if it submitted a false report that was later disputed,
+  the bond is transferred to the outsider.
```

Comment:
```suggestion
  report, confirmed as true by dispute or undisputed. If the oracle failed to
  provide a report, any account can report the outcome as an outsider.
  If the oracle submitted a report, which is not equal to the dispute result,
  the oracle bond is slashed. It is also slashed in case the oracle failed to report.
  The oracle bond is transferred to the outsider in the case, that the oracle failed
  to report and there was no dispute. If there was a dispute and the outsider turned
  out to be correct, the oracle bond is transferred to the outsider too. If the outsider
  was not correct and the market was disputed, the oracle bond is transferred to
  the account, which caused the dispute.
```

Reply:
What's the point of this change? Did I miss something?

Reply:
And I just have taken care of the fact that you use `outsider` without defining it beforehand.

Reply:
1ffc7147027c77430266d1e992447657c2e07244